media-gfx/opencsg/opencsg-1.5.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+MY_P="OpenCSG-${PV}"
+
+DESCRIPTION="The Constructive Solid Geometry rendering library"
+HOMEPAGE="https://www.opencsg.org"
+SRC_URI="https://www.opencsg.org/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~riscv ~x86"
+IUSE="doc"
+RESTRICT="test"
+
+RDEPEND="
+	dev-libs/libbsd
+	media-libs/glew:0=
+	virtual/opengl
+	x11-libs/libX11
+	x11-libs/libXau
+	x11-libs/libXdmcp
+	x11-libs/libXext
+	x11-libs/libxcb:=
+"
```

Comment:
```
 * VDB: detected possibly incorrect RDEPEND (media-gfx/opencsg-1.5.0)
 * dev-libs/libbsd   < 
 * x11-libs/libX11   < 
 * x11-libs/libXau   < 
 * x11-libs/libXdmcp < 
 * x11-libs/libXext  < 
 * x11-libs/libxcb:= < 
```
Could you check? It means these libraries aren't directly linked to 
```
# scanelf -n /usr/lib64/libopencsg.so.1.5.0
 TYPE   NEEDED FILE 
ET_DYN libGLEW.so.2.2,libGL.so.1,libstdc++.so.6,libgcc_s.so.1,libc.so.6 /usr/lib64/libopencsg.so.1.5.0
```

Reply:
I'm gonna verify those

Reply:
They were indeed not needed to build the package.